server/controllers/profile.js

```diff
@@ -6,119 +6,143 @@ const generateToken = require("../utils/generateToken");
 // @desc Given parameters passed in create a profile
 // @access Private
 
- exports.createProfile = asyncHandler(async (req, res, next) => {
-
-    const { firstName, lastName, description, email, phoneNumber, availability:{startDate,endDate,daysOfWeek} } = req.query;
-    
-    const profile = await Profile.create({
-        firstName,
-        lastName,
-        description,
-        email,
-        phoneNumber,
-        availability:{
-          startDate,
-          endDate,
-          daysOfWeek: []
-      }
-      });
-    
-      if (profile) {
-        const token = generateToken(profile._id);
-        const secondsInWeek = 604800;
-    
-        res.cookie("token", token, {
-          httpOnly: true,
-          maxAge: secondsInWeek * 1000
-        });
-    
-        res.status(201).json({
-          success: {
-            user: {
-              id: profile._id,
-              ffirstName: profile.firstName,
-              lastName: profile.lastName,
-              description: profile.description,
-              email: profile.email,
-              phoneNumber: profile.phoneNumber,
-              availability:{
-                startDate: profile.availability.startDate,
-                endDate: profile.availability.endDate,
-                daysOfWeek: []
-            }
-            }
-          }
-        });
-      } else {
-        res.status(400);
-        throw new Error("Invalid user data");
-      }
-})
- 
+exports.createProfile = asyncHandler(async (req, res, next) => {
+  console.log(req.body);
+  const new_obj = Object.assign(
+    {
+      availability: {
+        startDate: "2012-04-23T18:25:43.511Z",
+        endDate: "2012-04-23T18:25:43.511Z",
+        daysOfWeek: ["Saturday", "Monday"],
+      },
+    },
+    req.body
+  );
+  console.log(new_obj);
```

Comment:
Please remove `console` statements here. 
Do we need the enter an availability manually as a temporary solution? Can we not read it from request (e.g. therefore it could be `undefined` if its not yet implemented)? If not, can we please leave a comment reminder to change this?

Reply:
Sure availability field is yet to be implemented from other page, hence I hardcoded  here.

Reply:
@rajivtitus  doing it part of integration. Is it ok ?

Reply:
Yes, you can do this as a part of integration. You can exclude hardcoding it here and just leave a comment as a TODO item